Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -253,6 +268,18 @@ open class MessageContentCell: MessageCollectionViewCell {
         cellTopLabel.frame = CGRect(origin: .zero, size: attributes.cellTopLabelSize)
     }
     
+    /// Positions the cell's bottom label.
+    /// - attributes: The `MessagesCollectionViewLayoutAttributes` for the cell.
+    open func layoutCellBottomLabel(with attributes: MessagesCollectionViewLayoutAttributes) {
+        cellBottomLabel.textAlignment = attributes.cellBottomLabelAlignment.textAlignment
+        cellBottomLabel.textInsets = attributes.cellBottomLabelAlignment.textInsets
+        
+        let y = messageContainerView.frame.maxY + attributes.messageContainerPadding.bottom + attributes.messageBottomLabelSize.height
```

Comment:
For calculating the Y position, can we make it so the `cellBottomLabel` always starts after the `messageBottomLabel` max Y?

Reply:
I think it's better if the calculation of the layout of one cell does not depend on the layout of another cell. 
For example, the layout of `messageBottomLabel` is not calculated yet. if we gonna use it to calculate the layout of `cellBottomLabel`, we get the wrong layout of `cellBottomLabel`.
We have to agree either to use a certain order of the calculation of labels' layouts in `apply(_:)` function, or depend only on `messageContainerView`s layout in the calculations of labels' layouts.

Reply:
Fwiw, the rest of the layout methods rely on a certain order of method calls. I think making them dependent is a more reliable and easier way to do the layout

Reply:
Ok, did the fixes.